src/license.rs

```diff
@@ -0,0 +1,76 @@
+//! Copy `LICENSE` file(s) for the packaged wasm.
+
+use failure;
+use std::fs;
+use std::path::Path;
+
+use emoji;
+use glob::glob;
+use manifest;
+use progressbar::Step;
+use PBAR;
+
+fn get_license(path: &Path) -> Option<String> {
+    match manifest::get_crate_license(path) {
+        Ok(license) => license,
+        Err(_) => None,
+    }
+}
+
+fn glob_license_files(path: &Path) -> Result<Vec<String>, failure::Error> {
+    let mut license_files: Vec<String> = Vec::new();
+    for entry in glob(path.join("LICENSE*").to_str().unwrap())? {
+        match entry {
+            Ok(globed_path) => {
+                license_files.push(String::from(
+                    globed_path.file_name().unwrap().to_str().unwrap(),
+                ));
+            }
+            Err(e) => println!("{:?}", e),
```

Comment:
Instead of matching on the entry and ignoring errors, we should do something like this:

```rust
let path = entry?;
```

Reply:
Updated to propagate errors if any happen!